src/Aquifer.API/Endpoints/Resources/Content/SendForPublisherReview/Community/Endpoint.cs

```diff
@@ -0,0 +1,60 @@
+using Aquifer.API.Common;
+using Aquifer.API.Services;
+using Aquifer.Data;
+using Aquifer.Data.Entities;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.API.Endpoints.Resources.Content.SendForPublisherReview.Community;
+
+public class Endpoint(AquiferDbContext dbContext, IUserService userService, IResourceHistoryService historyService) 
+    : Endpoint<Request, Response>
+{
+    public override void Configure()
+    {
+        Post("/resources/content/{ContentId}/send-for-publisher-review-community");
+        Permissions(PermissionName.SendReviewCommunityContent);
+    }
+
+    public override async Task HandleAsync(Request request, CancellationToken ct)
+    {
+        var user = await userService.GetUserFromJwtAsync(ct);
+        var contentVersion = await dbContext.ResourceContentVersions
+            .Where(x => x.ResourceContentId == request.ContentId && x.AssignedUserId == user.Id && x.IsDraft)
+            .Include(x => x.ResourceContent)
+            .SingleAsync(ct);
+
+        if (contentVersion is null)
+        {
+            ThrowError("Content not found");
+        }
+
+        if (contentVersion.ResourceContent.Status != ResourceContentStatus.TranslationInProgress)
+        {
+            ThrowError("Content not in correct status");
```

Comment:
As above, I'd return a bad request here with this message, rather than blow up and force the 500.

Reply:
Does this not send a 400? The IDE context says `ThrowError` is:
> add a "GeneralError" to the validation failure list and send back a 400 bad request with error details immediately interrupting handler execution flow. if there are any vallidation failures, no execution will continue past this call.

Reply:
Okay, I assumed it was sending a 500 since it was an error, so sounds like it's equivalent to the 400 then.

Reply:
I think the only thing to consider is if this would trigger a log to app insights that could be alerted. Because it's not really an error, so much as a state we're just not allowing, so it's not something that should be throwing an exception under the hood. It does throw a `ValidationFailureException`, so we should verify if this is the correct flow (the Validators might already do the exact same thing).

Reply:
Yes, however, to me, if this is how the docs for fast endpoints says to return validation errors I would wonder why we would be alerting this in App Insights at all. It seems overly verbose for App Insights to alert on 400 errors. logging them, yes, but alerting, maybe not.